Makefile

```diff
@@ -49,7 +49,8 @@ STANDARDTESTFLAGS := -logtostderr
 TESTFLAGS         :=
 
 ifeq ($(STATIC),1)
-GOFLAGS  += -a -tags netgo -ldflags '-extldflags "-lm -lstdc++ -static"'
+GOFLAGS  += -tags netgo
```

Comment:
I seem to recall reading that `-tags netgo` didn't do anything in go1.4 because go1.4 won't rebuild the standard library. Have you verified this is still working?